server/database/database.js

```diff
@@ -15,7 +15,7 @@ const UserSchema = mongoose.Schema({
   username: String,
   pass: String,
   email: String,
-  subscriptions: [String],
+  subscriptions: [],
```

Comment:
Shouldn't this be `subscriptions: Array,`?

Reply:
That could work too. If you put [] or Array it is the same thing to mongoose. 

Note: specifying an empty array is equivalent to Mixed. The following all create arrays of Mixed:

const Empty1 = new Schema({ any: [] });
const Empty2 = new Schema({ any: Array });
const Empty3 = new Schema({ any: [Schema.Types.Mixed] });
const Empty4 = new Schema({ any: [{}] });